bbn-test-5/babylon-node/upgrades/v3rc0/README.md

```diff
@@ -0,0 +1,40 @@
+# `v3rc0` Software Upgrade
+
+This document summarizes the procedure to upgrade a Babylon node to version
+`v3.0.0-rc.0`.
+
+## Table of Contents
+
+1. [Overview](#1-overview)
+2. [Applying the upgrade](#2-applying-the-upgrade)
+   1. [Preparation](#21-preparation)
+   2. [Execution](#22-execution)
+
+## 1. Overview
+
+- **Upgrade version**: `v3rc0`
```

Comment:
While the binary version is going to be v3rc0, the upgrade's name is v3 ([ref](https://github.com/babylonlabs-io/babylon/blob/release/v3.x/app/upgrades/v3/upgrades.go#L21)). Which one should we use here?